x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -860,6 +860,7 @@ type KbnConfigSchemaNonOptionalProps<Props extends Record<string, unknown>> = Pi
  */
 export interface PolicyConfig {
   windows: {
+    advanced?: {};
```

Comment:
Are you getting any errors when you try to store data here?
Wondering if the type should be:

```suggestion
    advanced?: Record<string, string>;
```